tests/test_param_resource.py

```diff
@@ -31,3 +34,17 @@ def test_param_resource_override_name():
     resource = DemoPage()
     args = resource.parse_args(request)
     assert args["def"] is True
+
+
+def test_param_resource_error():
+
+    request = DummyRequest("/")
+    request.addArg(b"id", b"1234")
+    request.addArg(b"uuid", b"1234")
+
+    resource = DemoPage()
+
+    with pytest.raises(Error) as exc_info:
+        resource.parse_args(request)
+
+    assert exc_info.value.message == b"Error in parameter uuid: Malformed UUID"
```

Comment:
This is potentially a fragile test … It will break if we reformulate the error message, even if there's no bug, so you should think if it's worth having.

Reply:
Refactored the error message as a class attribute and reused it in the test.